validator/keymanager/remote-web3signer/client.go

```diff
@@ -0,0 +1,177 @@
+package remote_web3signer
+
+import (
+	"bytes"
+	"encoding/hex"
+	"encoding/json"
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/crypto/bls"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+)
+
+const (
+	ethApiNamespace = "/api/v1/eth2/sign/"
+	maxTimeout      = 5 * time.Second
+)
+
+type HTTPClient interface {
+	Do(req *http.Request) (*http.Response, error)
+}
+
+type client struct {
+	BasePath   string
+	restClient HTTPClient
+}
+
+func newClient(endpoint string) (*client, error) {
+	u, err := url.Parse(endpoint)
+	if err != nil {
+		return nil, errors.Wrap(err, "Invalid Format, unable to parse url")
```

Comment:
Errors are typically lowercase unless they are top-level logs
```
log.Error("Could not do %v", err)
```
```suggestion
		return nil, errors.Wrap(err, "invalid Format, unable to parse url")
```